systemd-swap

```diff
@@ -444,7 +454,11 @@ gen_zram_device(){
                                 INFO "Zram: swapoff $DEV"
                                 swapoff "$DEV" &
                         fi
-                        wait
+                done
+
+                wait ${!}
```

Comment:
You fork several job, but wait only for latest.
I think that's better to just use bare wait.
Because we can't ensure what latest job will finish only after all others.

Reply:
Sorry have zero experience with bash, Google said it will wait for everything. What about other lines?

Reply:
wait with arg, will wait for arg.